### ISSUE TITLE
eval.py execution

### ISSUE THREAD

**Issue by kaushikpasi** [dennybritz/cnn-text-classification-tf#27]
I trained the latest code with default parameters and when executing the same eval.py with the proper directory name, I am getting this output:

Parameters:
ALLOW_SOFT_PLACEMENT=True
BATCH_SIZE=64
CHECKPOINT_DIR=./runs/1466501212/checkpoints/
LOG_DEVICE_PLACEMENT=False

Evaluating...

Total number of test examples: 2
Accuracy: 0

Why am I getting accuracy as '0'?

**Comment by RachelDavis** [comment]
Hello! I just got the code updated 10 days ago and I'm getting a 50% accuracy on a test set of 2 examples. I'm wondering why the test set is only 2 examples now, as the dataset was significantly larger.

**Comment by dennybritz** [comment]
The change was to make it easier to evaluate your own data by modifying the eval.py. I added back the option to evaluate on all training data in https://github.com/dennybritz/cnn-text-classification-tf/commit/6dbd02a2b227b8cb442793eb405a3889d32c623c using this:

```
./eval.py --eval_train --checkpoint_dir=...
```
